# This patch was adapted from the one on Arch Linux:
https://gitlab.archlinux.org/archlinux/packaging/packages/opensnitch/-/blob/main/add-go.sum-file.patch
--- a/daemon/go.mod
+++ b/daemon/go.mod
@@ -9,10 +9,16 @@ require (
 	github.com/google/nftables v0.1.0
 	github.com/google/uuid v1.3.0
 	github.com/iovisor/gobpf v0.2.0
+	github.com/josharian/native v0.0.0-20200817173448-b6b71def0850
+	github.com/mdlayher/netlink v1.4.2
+	github.com/mdlayher/socket v0.0.0-20211102153432-57e3fa563ecb
 	github.com/varlink/go v0.4.0
 	github.com/vishvananda/netlink v0.0.0-20210811191823-e1a867c6b452
+	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae
 	golang.org/x/net v0.0.0-20211209124913-491a49abca63
 	golang.org/x/sys v0.0.0-20211205182925-97ca703d548d
+	golang.org/x/text v0.3.6
+	google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
 	google.golang.org/grpc v1.32.0
-	google.golang.org/protobuf v1.26.0 // indirect
+	google.golang.org/protobuf v1.26.0
 )
